open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/RelationshipsFVT.java

```diff
@@ -1423,4 +1432,52 @@ private ProjectScope createProjectScope(Project project, Term term) throws Inval
         System.out.println("CreatedProjectScopeRelationship " + createdProjectScope);
         return createdProjectScope;
     }
+
+    private void categoryHierarchyLinkFVT(Category parent, Category child) throws UserNotAuthorizedException, PropertyServerException, InvalidParameterException, SubjectAreaFVTCheckedException {
+        CategoryHierarchyLink categoryHierarchyLink = createCategoryHierarchyLink(parent, child);
+        String guid = categoryHierarchyLink.getGuid();
+        CategoryHierarchyLink gotCategoryHierarchyLink = subjectAreaRelationship.categoryHierarchyLink().getByGUID(this.userId, guid);
+        FVTUtils.validateLine(gotCategoryHierarchyLink);
+        System.out.println("Got CategoryHierarchyLink " + categoryHierarchyLink);
+        Category gotChild = subjectAreaCategory.getByGUID(userId, child.getSystemAttributes().getGUID());
+        checkParent(parent, gotChild);
+        subjectAreaRelationship.categoryHierarchyLink().delete(this.userId, guid);
+        //FVTUtils.validateLine(gotTermCategorizationRelationship);
+        System.out.println("Soft deleted CategoryHierarchyLink with userId=" + guid);
+        gotCategoryHierarchyLink = subjectAreaRelationship.categoryHierarchyLink().restore(this.userId, guid);
+        FVTUtils.validateLine(gotCategoryHierarchyLink);
+        System.out.println("Restored CategoryHierarchyLink with userId=" + guid);
+        subjectAreaRelationship.categoryHierarchyLink().delete(this.userId, guid);
+        //FVTUtils.validateLine(gotTermCategorizationRelationship);
```

Comment:
I suggest we remove the commented out line of code we are adding

Reply:
I agree to you